components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -106,7 +108,7 @@ fun Context.share(text: String, subject: String = getString(R.string.mozac_suppo
  * (via https://stackoverflow.com/a/12362545/512580)
  */
 inline val Context.isScreenReaderEnabled: Boolean
-    get() = (getSystemService(Context.ACCESSIBILITY_SERVICE) as AccessibilityManager).isTouchExplorationEnabled
+    get() = getSystemService<AccessibilityManager>()!!.isTouchExplorationEnabled
```

Comment:
nit: let's try to reduce code cleanup noise for unrelated things in general when fixing a bug or adding a feature. It makes it hard for the reviewer to know what is related to the change and what isn't.